GameData/KerbalismConfig/System/Parts.cfg

```diff
@@ -1062,13 +1062,14 @@
         id_value = _AirPump
       }
 
-      RESOURCE
-      {
-        name = Nitrogen
-        amount = #$/CrewCapacity$
-        maxAmount = #$/CrewCapacity$
-        @amount *= 500  // FIXME 
-        @maxAmount *= 500
+		//Nitrogen tanks are in the SM/ESM, not the capsule
+      //RESOURCE
+      //{
+        //name = Nitrogen
+        //amount = #$/CrewCapacity$
+        //maxAmount = #$/CrewCapacity$
+        //@amount *= 500  // FIXME 
+        //@maxAmount *= 500
```

Comment:
You can delete the entire section, no need to keep it if the capsule shouldn't have any tank.